src/css/base.scss

```diff
@@ -183,16 +200,35 @@ body {
 
 .total-spent {
   padding: 15px;
-  color: red;
+  color: black;
   font-size: 1.1em;
-  background-color: #678464;
+  background-color: #EDE7B1;
   width: 60%;
-  height: 40%;
-  border-radius: 15px;
+  border-radius: 5px;
   border: solid 1px black;
   margin-top: 10px;
 }
 
+.filter-by {
+  text-decoration: underline;
+  font-weight: bolder;
+  font-size: 1.3em;
+  margin-bottom: 10px;
+}
+
+.filter-buttons.approve-button {
+  font-size: 0.8em;
+  border-radius: 5px;
+  padding: 10px;
+  margin-top: 10px;
```

Comment:
I see a few examples of 10px for margin and padding, this could maybe be a mixin if you use both in several spots, or you could have a variable called something like $standard-padding or $standard-margin